src/databricks/labs/ucx/source_code/known.json

```diff
@@ -1278,6 +1278,11 @@
   "contourpy": {
     "contourpy": []
   },
+  "core": {
```

Comment:
What kind of package is this?

Reply:
I believe it's this one: https://pypi.org/project/python-core/

Reply:
can you add it again as a comment? It is a weird package, no info at all

Reply:
As already mentioned, json does not support comments

Reply:
ah missed that, that is unfortunate

Reply:
https://pypi.org/project/python-core/ does not look like the right package, contents is as follows:

<img width="385" alt="Screenshot 2024-07-11 at 15 12 55" src="https://github.com/databrickslabs/ucx/assets/5946784/22b8b056-384b-4b19-a35a-ea793f7b3b74">

Reply:
Actually it comes from https://pypi.org/project/dbl-sat-sdk/0.1.34/

Reply:
this is noise package